### DISCUSSION TITLE
Have the WritePermissions option mentioned in the docs

### DISCUSSION THREAD

**Discussion by maxkratt** [FirstGearGames/FishNet#388] [Documentation]
Perhaps the SyncVarAttribute `WritePermissions` could be mentioned in the following docs page: https://fish-networking.gitbook.io/docs/manual/guides/synchronizing/attributes
And also described on the API page here: https://firstgeargames.com/FishNet/api/api/FishNet.Object.Synchronizing.SyncVarAttribute.html

**Comment by FirstGearGames** [comment]
This discussion is still valid but the document page is going to look a bit different with the v4 changes.

**Comment by FirstGearGames** [comment]
No longer applicable. SyncTypeSettings is noted and that API will show capabilities.
